pkgs/applications/networking/instant-messengers/teams-for-linux/default.nix

```
@@ -91,12 +93,13 @@ stdenv.mkDerivation (finalAttrs: {
   })];
 
   passthru.updateScript = ./update.sh;
+  passthru.tests.version = testers.testVersion { package = teams-for-linux; };
```

```suggestion
  passthru.tests.version = testers.testVersion rec {
    package = teams-for-linux;
    command = "HOME=$TMPDIR ${package.meta.mainProgram or package.pname} --version";
  };
```

Okay so it turns out doing `teams-for-linux --version` expects a writable `HOME` set to be able to work and currently it fails

So this diff should do it, and then this PR is good to go. Apologies for the back and forth, but thank you again for working on this!